config/envs/rhdm-trial.yaml

```diff
@@ -7,6 +7,11 @@ console:
           spec:
             containers:
               - name: "[[.ApplicationName]]-[[.Console.Name]]"
+                env:
+                  - name: KIE_SERVER_HOST
```

Comment:
What is the purpose of specifying KIE_SERVER_HOST into console deployment config?

Reply:
Setting it without value will override the value from the common.yaml file

Reply:
Ok, thanks

Reply:
But why is it specified for Workbench? AFAIK Workbench has nothing to do with KIE_SERVER_HOST.

Reply:
Because it's wrong and shouldn't be there :wink: